hcl2template/plugin.go

```diff
@@ -54,7 +54,18 @@ func (cfg *PackerConfig) PluginRequirements() (plugingetter.Requirements, hcl.Di
 	return reqs, diags
 }
 
-func (cfg *PackerConfig) DetectPluginBinaries() hcl.Diagnostics {
+func (cfg *PackerConfig) DetectPluginBinaries(releaseOnly bool) hcl.Diagnostics {
```

Comment:
Given the fact that this boolean to only load official release binaries is being passed into the plugin config methods what do you think about adding this as a field to the PluginConfig types as opposed to changing the function signature for the discover methods?

Reply:
I think that this would position us better to support environment variables to control this flag and also possible loading of a plugin config from some sort of rc file in the future.

Reply:
That's a good call, I'll see if it is possible to add this elegantly!